brain_games/game.py

```diff
@@ -0,0 +1,21 @@
+from prompt import string
```

Comment:
this module is not a game, but rather something that executes games. what name could be more appropriate?